src/commandDetails/coin/check.ts

```diff
@@ -13,12 +13,11 @@ import { getCoinEmoji } from '../../components/emojis';
 // Check a user's balance
 const coinCheckExecuteCommand: SapphireMessageExecuteType = async (
   _client,
-  _messageFromUser,
+  messageFromUser,
   args
 ): Promise<SapphireMessageResponse> => {
-  // Mandatory argument is user
-  const user = <User>args['user'];
-
+  // use the caller as a default user if no argument is provided
+  const user = <User>args['user'] ?? messageFromUser.member?.user;
```

Comment:
Does `messageFromUser.user` work?

Reply:
Nope, `user` is not a valid field for a `Message`. `author` exists for message, but I don't really know in what situation accessing member could be null for a message 🤷 so it's probably not a concern